README.md

```diff
@@ -115,6 +115,24 @@ A full list of all the unit tests we run in our CI can be found [here](.github/s
 MMTk is also tested with the VM bindings we are maintaining by running standard test/benchmark suites for the VMs.
 For details, please refer to each VM binding repository.
 
+## Supported Rust versions
+
+MMTk uses a pinned Rust version in the repository (recorded in the `rust-toolchain` file).  We run
+our tests and benchmarks using the pinned Rust version. We recommend using the pinned Rust version
+for development. We update the pinned Rust version between releases of mmtk-core to keep it close to
+the latest Rust stable release. The release cycle of mmtk-core is six weeks, roughly the same as
+Rust itself. 
+
+Our minimum support Rust version (MSRV) policy is "N-1".  That means we also ensure mmtk-core works
```

Comment:
I think it's worth reminding that the N is NOT the current Rust release. N is the version recorded in `rust-toolchain`, which might lag behind the current stable Rust stable release.